typescript/cli/src/commands/signCommands.ts

```diff
@@ -6,6 +6,8 @@ export const SIGN_COMMANDS = ['deploy', 'send'];
 export function isSignCommand(argv: any): boolean {
   return (
     SIGN_COMMANDS.includes(argv._[0]) ||
-    (argv._.length > 1 && SIGN_COMMANDS.includes(argv._[1]))
+    (argv._.length > 1 &&
+      (SIGN_COMMANDS.includes(argv._[1]) ||
+        (argv._[1] === 'avs' && argv._[2] === 'register')))
```

Comment:
why not add this to the SIGN_COMMANDS const instead?

Reply:
needs argv[0] and argv[1].